modules/aggregation-layers/src/screen-grid-layer/screen-grid-layer.js

```diff
@@ -53,16 +50,19 @@ const defaultProps = {
 
 export default class ScreenGridLayer extends Layer {
   getShaders() {
-    const shaders = isWebGL2(this.context.gl) ? {vs, fs} : {vs: vs_WebGL1, fs: fs_WebGL1};
-    shaders.modules = ['picking'];
-    return super.getShaders(shaders);
+    return super.getShaders({vs, fs, modules: ['picking']});
   }
 
   initializeState() {
     const attributeManager = this.getAttributeManager();
     const {gl} = this.context;
+    if (!hasFeatures(gl, [FEATURES.TEXTURE_FLOAT])) {
+      // max aggregated value is sampled from a float texture
+      this.setState({supported: false});
+      log.error(`ScreenGridLayer: ${this.id} is not supported on this browser`)();
```

Comment:
Wait what happened to the CPU aggregation support?

Reply:
This error is produced only when it is WebGL1 and `OES_texture_float` is not supported. This extension is widely supported (including Safari). I have link to stats in the issue.

Reply:
To elaborate, we still support CPU Aggregation path (Safari). Before these changes, we supported even when `OES_texture_float` is not supported, but now we don't. But cases where this extension is not supported seem to be [limited to IE](https://developer.mozilla.org/en-US/docs/Web/API/OES_texture_float).